content/articles/domains-uk.markdown

```diff
@@ -61,11 +61,66 @@ Instead of creating a new contact in DNSimple, it's perfectly fine to reuse an e
 
 ## Transferring a .UK domain to DNSimple {#transfer}
 
-To transfer a UK domain (i.e. example.uk or example.co.uk) into DNSimple, follow [these instructions](/articles/transferring-uk-domain).
+`.UK` and `.CO.UK` domains have a different transfer process than most other domain registrations. These steps explains some of those differences.
+
+### Before you start
+
+<warning>
+It is important you follow these steps in strict order, other case, there is a risk of loosing your domains registration.
+</warning>
+
+<warning>
+As with any domain transfer, to avoid the possibility of downtime you should [point the name servers for the domain to DNSimple (or another provider) before you begin the transfer](/articles/before-transferring-domain). This will prevent downtime and will also allow you to perform changes to the DNS records during the domain transfer.
+</warning>
+
+<div class="section-steps" markdown="1">
+##### 1. Starting the transfer
+
+1. Log into DNSimple and click on <label>Add a Domain</label>.
+1. Select <label>transfer a domain</label>.
+1. Press <label>Transfer Domain</label>
+1. Select a registrant and fill in any extended attributes.
+1. Press <label>Transfer Domain</label>
+</div>
+
+<div class="section-steps" markdown="1">
+##### 2. Approving the transfer
+
+To complete the transfer of a `.UK` or `.CO.UK` domain, you must change the domain's IPS TAG at your current domain registrar. Your registrar may allow you to do this through their website, or you may need to contact them via email or phone.
```

Comment:
Would it be worth it here to add another line about only doing this after you've selected the domain to transfer? What do you think, @weppos ?

Reply:
I think it's enough with the <warning> paragraph in the previous section, but I'm open to add it :)

Reply:
The article seems to be clear that you first need step 1 then step 2. I will keep it as it is for now.